src/app/component/CookiePopup/CookiePopup.component.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import { PureComponent } from 'react';
+import BrowserDatabase from 'Util/BrowserDatabase';
+import Link from 'Component/Link';
+import './CookiePopup.style';
+
+export const COOKIE_POPUP = 'cookie_popup';
+
+class CookiePopup extends PureComponent {
+	constructor() {
+		super();
+
+		this.state = {
+			isAccepted: BrowserDatabase.getItem(COOKIE_POPUP) || false
+		};
+
+		this.acceptCookies = this.acceptCookies.bind(this);
+	}
+
+	acceptCookies() {
+		BrowserDatabase.setItem(true, COOKIE_POPUP);
+		this.setState({ isAccepted: true });
+	}
+
+  	render() {
+		const { isAccepted } = this.state;
+
+		return (
+			<div block="CookiePopup" mods={ { isAccepted: isAccepted } }>
+				<div block="CookiePopup" elem="Content">
+					<p>
+						{ __('We use cookies, also from third parties, to remember your settings, statistics and targeted marketing. You accept all cookies by clicking further or by clicking "accept". Read more about our cookies and their purpose in our Cookie Policy - ') }
```

Comment:
I think this also can we taken from Magento config, but I am not sure.

Reply:
The text is not possible as default to add from magento.
There is 2 thing we can do tho, either we make a rich text editor in the cookie settings section and make graphql support, or we can make it as a block.

It's up to you what you prefer?

Reply:
Let's add the `cookie notice` configuration filed (text) not supporting HTML in *Design / Configuration*. You can do this in [store-graphql](https://github.com/scandipwa/store-graphql). Expose it via GraphQL `storeConfig` type :) Thanks!